src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -1542,3 +1542,1398 @@
+        - name: add a hybrid connection to the app
+          text: >
+              az webapp hybrid-connection add -g [myResourceGroup] -n [myAppName]
+              --namespace [hybridConnectionNamespaceName] --hybrid-connection [hybridConnectionName]
+        - name: remove the hybrid connection from the app
+          text: >
+              az webapp hybrid-connection remove -g [myResourceGroup] -n [myAppName]
+              --namespace [hybridConnectionNamespace] --hybrid-connection [hybridConnectionName]
+"""
+
+helps['webapp hybrid-connection list'] = """
+    type: command
+    short-summary: lists all the hybrid connections set on the specified app
+    examples:
+        - name: list the hybrid connections
+          text: >
+              az webapp hybrid-connection list -g [myResourceGroup} -n {myAppName]
+"""
+
+helps['webapp hybrid-connection add'] = """
+    type: command
+    short-summary: adds an existing hybrid connection to the specified app
+    parameters:
+        - name: --hybrid-connection
+          type: string
+          short-summary: 'Hybrid connection name'
+        - name: --namespace
+          type: string
+          short-summary: 'Hybrid connection namespace'
+    examples:
+        - name: add a hybrid connection to the app
+          text: >
+            az webapp hybrid-connection add -g [myResourceGroup] -n [myAppName]
+            --namespace [hybridConnectionNamespaceName] --hybrid-connection [hybridConnectionName]
+"""
+
+helps['webapp hybrid-connection remove'] = """
+    type: command
+    short-summary: disconnects the specified hybrid connection from the specified app
+    parameters:
+        - name: --hybrid-connection
+          type: string
+          short-summary: 'Hybrid connection name'
+        - name: --namespace
+          type: string
+          short-summary: 'Hybrid connection namespace'
+    examples:
+        - name: remove the hybrid connection from the app
+          text: >
+            az webapp hybrid-connection remove -g [myResourceGroup] -n [myAppName]
+            --namespace [hybridConnectionNamespace] --hybrid-connection [hybridConnectionName]
+"""
+
+helps['appservice hybrid-connection'] = """
+    type: group
+    short-summary: commands that interact with hybrid connections on an app service level - existing command sets which key each app uses to connect to the hybric connection
+    examples:
+        - name: set the key the apps should use
+          text: >
+            az appservice hybrid-connection set-key --asp [hybridConnectionAppServicePlan] -g [appServicePlanResourceGroup]
+            --namespace [hybridConnectionNamespace] --hybrid-connection [hybridConnectionName]
+            --key-type ["primary" | "secondary"]
+"""
+
+helps['appservice hybrid-connection set-key'] = """
+    type: command
+    short-summary: specifies which key all the apps using a hybrid connection should use when connecting to it
+    parameters:
```

Comment:
Generic: keep the help consistent with the rest, some of them have params some don't.